src/sonic-config-engine/minigraph.py

```diff
@@ -1616,7 +1616,7 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         'config': {
             'client_auth': 'true',
             'allow_insecure': 'false',
-            'log_level': 'trace'
+            'log_level': 'info'
```

Comment:
can we have it as notice level? Also could you place in description how to enable the log to info/trace if it is required for a debugging?